tests/integration/suite/actors/metadata/actor_client.go

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Remove `actor_` prefix to files in directory as they are superfluous being in the `actors` top level suite category.

Reply:
Done